src/components/z-link.scss

```diff
@@ -0,0 +1,152 @@
+:root {
+  --font-size-link: inherit;
+  --font-weight-link: inherit;
+}
+
+a.z-link {
+  display: inline-flex;
+  color: var(--color-link-primary);
+  fill: var(--color-link-primary);
+  cursor: pointer;
+  font-family: var(--font-family-sans);
+  font-size: var(--font-size-link);
+  font-weight: var(--font-weight-link);
+  line-height: inherit;
+  text-decoration: none;
+}
+
+/* LINK HOVER AND FOCUS Status */
+a:hover.z-link,
+a:focus.z-link {
+  color: var(--color-hover-link);
+  fill: var(--color-hover-link);
+}
+
+a:hover.z-link,
+a:focus.z-link {
+  text-decoration: underline;
+}
+
+/* LINK PRESSED Status */
+a:active.z-link {
+  color: var(--color-pressed-link);
+  fill: var(--color-pressed-link);
+  text-decoration: underline;
```

Comment:
questo lo metterei nella regola precedente, in modo che poi qui restino solo le cose che sono effettivamente solo dell' :active